crates/bitwarden-ssh/src/models.rs

```diff
@@ -0,0 +1,5 @@
+pub struct SshKey {
+    pub private_key: String,
+    pub public_key: String,
+    pub key_fingerprint: String,
+}
```

Comment:
**suggestion**: I think the naming here can be confusing. We already have a concept of `SshKey`, and they are not the same type. We can either re-use `SshKeyView`, or change this to be a DTO, we generally call these the same as the function and append result, i.e. `GenerateKeypairResult`.

Reply:
With a DTO we can also lift this into the lib.rs to keep it closer to the function producing it.